activity.js

```diff
@@ -2,17 +2,32 @@ class Activity {
   constructor(category, description, minutes, seconds, completed) {
     this.category = category;
     this.description = description;
-    this.minutes = minutes;
-    this.seconds = seconds;
+    this.minutes = parseInt(minutes);
+    this.seconds = parseInt(seconds);
```

Comment:
Glad we used this method here to get the number to be a 'number' and not a string!